### PR TITLE
Cfp

### PR DESCRIPTION
**Pull request by frenzymadness**
Changes I am able to do to open CfP tomorrow.

Next steps:

- [ ] redesign `btn-primary` buttons to use own brown color
- [ ] fix photo input
- [ ] fix form fields to use black color for text
- [ ] implement email notification
- [ ] split forms to two halfs: public & private info

### CODE REVIEW

File: pyconcz/proposals/models.py [hunk 1]
```diff
@@ -118,10 +119,11 @@ class Talk(EntryBase):
         upload_to='proposals/pyconcz2018/talks/',
```
Comment (hroncok) [inline pyconcz/proposals/models.py]: Careful with the path here. it says 2018.

File: pyconcz/proposals/models.py [hunk 2]
```diff
@@ -233,10 +236,11 @@ class Workshop(EntryBase):
...
```
Comment (hroncok) [inline pyconcz/proposals/models.py]: dtto

File: pyconcz/proposals/models.py (hunk 1 above)
Comment (frenzymadness) [reply pyconcz/proposals/models.py]: Fixed

File: pyconcz/proposals/models.py (hunk 2 above)
Comment (frenzymadness) [reply pyconcz/proposals/models.py]: Fixed

**Pull request merged**